pkgs/tools/security/gosh/default.nix

```
@@ -26,6 +26,6 @@ buildGoModule rec {
     description = "Reverse/bind shell generator";
     homepage = "https://github.com/redcode-labs/GoSH";
     license = licenses.mit;
-    maintainers = with maintainers; [ fab ];
+    maintainers = with maintainers; [ fab unrooted wr0belj wintrmvte ];
```

```suggestion
    maintainers = with maintainers; [ fab ] ++ teams.redcodelabs.members;
```

And same for other packages, otherwise there is actually no need to add a team in maintainers/team-list.nix :-)